hbase-testing-util/pom.xml

```diff
@@ -152,6 +152,19 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
```

Comment:
Why we need to add these two plugins here?

Reply:
We don't, it's just a lefotover from an eralier version.
Thanks for catching this, removed.